src/shogun2-core/src/main/java/de/terrestris/shogun2/importer/communication/RESTDataDirectory.java

```diff
@@ -0,0 +1,19 @@
+package de.terrestris.shogun2.importer.communication;
+
+import de.terrestris.shogun2.importer.communication.RESTData;
+
+/**
+ *
+ * @author Daniel Koch
+ * @author terrestris GmbH & Co. KG
+ *
+ */
+public class RESTDataDirectory extends RESTData {
```

Comment:
Is this class needed currently?

Reply:
No, actually not. I would leave it for future purposes, ok?

Reply:
Alright.